### PR TITLE
ENH: add OHBM23 educational talk

### PR DESCRIPTION
**Pull request by celprov**
Hey,
I would like to add my OHBM23 presentation in the list of educational resources available.
However, it's a PDF so it requires some tricks to get rendered.

I tried to follow the guidelines from [here](https://stackoverflow.com/questions/69237778/how-to-embed-a-local-pdf-file-in-mkdocs-generated-website-on-github-pages) without success so far.
Any idea?

Additionally, I was thinking to add a link to the recording that I will post on Youtube. WDYT @oesteban ?

### CODE REVIEW

**Comment by oesteban** [comment]
I don't get why you want to see the PDF embedded. I guess you want it to provide the same experience of the other slides, but those are not PDF; those are HTML.

I think you should just upload the pdf file (you did) and link it.

**Comment by celprov** [comment]
@oesteban ok that was easy to solve ^^'

File: docs/users/educational.md [hunk 1]
```diff
@@ -7,4 +7,7 @@
 
 ## QC protocols and Standard Operating Procedures
 
-- [MRIQC-SOPs, a template repository for version-controlled SOPs](https://github.com/mriqc-sops). The exemplar template is rendered [here](../../mriqc-sops)
\ No newline at end of file
+- [SOPs-cookiecutter, a template repository for version-controlled SOPs](https//github.com/nipreps/sops-cookiecutter). The exemplar template is rendered [here](../../mriqc-sops)
```
Comment (oesteban) [inline docs/users/educational.md:10]:
```suggestion
- [SOPs-cookiecutter, a template repository for version-controlled SOPs](https//github.com/nipreps/sops-cookiecutter). The example template is rendered [here](/sops-cookiecutter/)
```

**Review by oesteban — COMMENTED** [review]
One minor suggestion. Once that's fixed, please go ahead and merge.

**Comment by celprov** [comment]
@oesteban Thanks ! It is done. However, I don't have merging rights it seems. Can you do it?

**Pull request merged**